feat(monorepo): split indexer into apps and packages (#68)
namehash/ensnode: packages/ensnode-utils/README.md

# ENSNode Utils

> This package is a utility library for interacting with ENS (Ethereum Name Service) data. It contains various helper functions and tools to facilitate interactions with ENS nodes, such as name resolution, hashing, and other common operations related to ENS.

Reviewer: ```suggestion
This package is a utility library for interacting with ENS (Ethereum Name Service) data. It contains various helper functions and tools to facilitate interactions with ENS and ENSNode instances.
```
